project-base/docker/php-fpm-deployment/Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM php:7.2-fpm-alpine
```

Comment:
We should do this using multi-stage build (https://docs.docker.com/develop/develop-images/multistage-build/) because this "copying" of docker files leads to possible problems in the future.

Reply:
You can create multistage build only if you push base Dockerfile first, then you can extend it. This is something that is not implemented now and I think it is out of scope of this US

Reply:
Can you create new US then?